src/components/DpDraggable/DpDraggable.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpDraggable from './DpDraggable.vue'
+
+const meta: Meta<typeof DpDraggable> = {
+    component: DpDraggable,
+    title: "Components/Draggable",
+    render: (args) => ({
+        components: {
+            DpDraggable,
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-draggable v-bind="args">
```

Comment:
Should we syncronize this example code with the example in the mdx file?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/497/commits/089a7a49c5d37e64eb42122ba6901f6eb3cc4769

Reply:
now that you changed the other place, i'd update it here, too.

Reply:
i understand that it is challenging to implement real state within the story, so we can also leave it like so, for the time being.

Reply:
@spiess-demos https://github.com/demos-europe/demosplan-ui/pull/497/commits/c63541d7d3cf9737b4de5115f2529b21cf1e38a3